TheBTeam.BLL/Services/Edits.cs

```diff
@@ -0,0 +1,297 @@
+                            user.Age = tempuser.Age;
+                            user.Company = tempuser.Company;
+                            user.Gender = tempuser.Gender;
+                            user.Phone = tempuser.Phone;
+                            user.Currency = tempuser.Currency;
+                            user.Balance = tempuser.Balance;
+                            Console.WriteLine("\nSaved");
+                            choiceSubmitted = true;
+                            exit = true;
+                        }
+
+                        else if (key.ToUpper() == "X")
+                        {
+                            Console.WriteLine("\nCancelled");
+                            choiceSubmitted = true;
+                            exit = true;
+
+                        }
+                        else
+                        {
+                            Console.WriteLine("Select X to abort or S to save please\n");
+                        }
+                    }
+                }
+            }
+        }
+
+        public static void EditTransaction(Transaction transaction)
+        {
+            var tempTransaction = new Transaction(transaction.User, transaction.Type, transaction.Category, transaction.Currency, transaction.Amount);
+            string options = "1)Edit category\n2)Save or cancel changes";
+            bool exit = false;
+            while (exit == false)
+            {
+                Console.WriteLine(options, "\n");
+                string choice = Console.ReadLine();
+                if (choice == "1")
+                {
+                    tempTransaction.Category = GetCategoryOfTransaction();
+                }
+                else if (choice == "2")
+                {
+                    Console.WriteLine("S to save changes or X to cancel\n");
+                    bool choiceSubmitted = false;
+                    while (choiceSubmitted == false)
+                    {
+                        string key = Console.ReadKey().Key.ToString();
+                        if (key.ToUpper() == "S")
+                        {
+                            transaction.Category = tempTransaction.Category;
+                            choiceSubmitted = true;
+                            Console.WriteLine("\nSaved");
+                            exit = true;
+                        }
+                        else if (key.ToUpper() == "X")
+                        {
+                            choiceSubmitted = true;
+                            Console.WriteLine("\nCanceled");
+                            exit = true;
+                        }
+                        else
+                        {
+                            Console.WriteLine("Select S to save changes or X to cancel\n");
+                        }
+                    }
+                }
+            }
+        }
+
+        private static string GetStringInput(string name, int minLength)
```

Comment:
i made new method if PR will be ok, tyou ncan implement it

Reply:
ill remember that once its merged thanks